### PR TITLE
Cage

### PR DESCRIPTION
**Pull request by Sanyyouisf**

### CODE REVIEW

**Comment by Sanyyouisf** [comment]
@drdreed
finish

File: javascripts/main.js [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+
+    $("#btn").on("click", function() {
+        let input2 = $("#input2").val();
+        let input1 = $("#input1").val();
+       
+
+        const loadFirstPlayer = () => {
+            return new Promise((resolve, reject) => {
+                // $.ajax("https://teamtreehouse.com/" + input1 + ".json")
+                $.ajax("https://teamtreehouse.com/sanyyousif.json")
```
Comment (DevDreed) [inline javascripts/main.js]: This is always the same user.  It does not take from the input field.

File: javascripts/main.js [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+                    .done((data1) => resolve(data1))
+                    .fail((error) => reject(error));
+            });
+        };
+
+
+        const loadSecondPlayer = () => {
+            return new Promise((resolve, reject) => {
+                // $.ajax("https://teamtreehouse.com/" + input2 + ".json")
+                $.ajax("https://teamtreehouse.com/ellisthomas.json")
```
Comment (DevDreed) [inline javascripts/main.js]: Same thing here.

File: javascripts/main.js [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+        const playerPointsArray = [];
+        let winner = "";
+        const checkForWinner = (players) => {
+            playerPointsstrings = "";
+            for (let i = 0; i < players.length; i++) {
+                playerPointsstrings = players[i].points.total;
+                playerPointsArray.push(playerPointsstrings);
+            }
+            if (playerPointsArray[0] > playerPointsArray[1]) {
+                $("#winner").val((players[0].name)+ "winns");
```
Comment (DevDreed) [inline javascripts/main.js]: spelling on wins

**Review by DevDreed — CHANGES_REQUESTED** [review]
Please see the changes I have requested.

File: javascripts/main.js (hunk 3 above)
Comment (Sanyyouisf) [reply javascripts/main.js]: @drdreed
right I will change it .
thanks

**Comment by Sanyyouisf** [comment]
@drdreed
finish

**Comment by DevDreed** [comment]
👍

**Pull request merged**